### PR TITLE
Updated Victory Conditions for Imrik

### PR DESCRIPTION
**Pull request by BoyNamedHsu**
Changed building requirements and unit requirements.

Removed weird infantry requirements, replaced with dragons and the special quest dragons.
Also removed the weird building requirements, but added Dragon Keep to the other construct requirement list.
This fixes the replaced Handmaiden building in T&T update.

### CODE REVIEW

**Comment by msolefonte** [comment]
Seems amazing! I am not sure about having 5/15 Dragons though. Isn't it a bit too much? I usually play with antisnowball mods so doomstacks are strongly discouraged. Something like 4/8 or 4/10 may be enough to leave the message. What do you think?

Also, thank you very much for your PRs again. I tried to make this mod/framework as community-friendly as possible but no one pushed anything before.

**Review by msolefonte — CHANGES_REQUESTED** [review]
I'll change the Dragon unit requirements from 5/15 to 4/8. We can discuss it in the PR thread if you want to.

File: src/script/vco2_main/factions/wh2_dlc15_hef_imrik/campaigns/main_warhammer/missions.lua [hunk 1]
```diff
@@ -1,186 +1,160 @@
...
-				}
-				objective
-				{
-					type CONTROL_N_PROVINCES_INCLUDING;
-					total 18;
-				}
+                objective
+                {
+                    type OWN_N_UNITS;
+                    total 15;
```
Comment (msolefonte) [inline src/script/vco2_main/factions/wh2_dlc15_hef_imrik/campaigns/main_warhammer/missions.lua:133]: I'll change the Dragon unit requirements from 5/15 to 4/8. We can discuss it in the PR thread if you want to.

File: src/script/vco2_main/factions/wh2_dlc15_hef_imrik/campaigns/main_warhammer/missions.lua [hunk 2]
```diff
@@ -1,186 +1,160 @@
...
                     faction wh2_dlc15_hef_imrik;
-                    total 1;
+                    total 2;
                     building_level wh2_dlc15_special_graves_of_the_dragons_2_hef;
+                    building_level wh2_main_hef_dragons_2;
+                }
+                objective
+                {
+                    type OWN_N_UNITS;
+                    total 5;
```
Comment (msolefonte) [inline src/script/vco2_main/factions/wh2_dlc15_hef_imrik/campaigns/main_warhammer/missions.lua:52]: I'll change the Dragon unit requirements from 5/15 to 4/8. We can discuss it in the PR thread if you want to.

**Review by msolefonte — COMMENTED** [review]
I'll change the Dragon unit requirements from 5/15 to 4/8. We can discuss it in the PR thread if you want to.

**Comment by BoyNamedHsu** [comment]
Yeah, that was the thread I was looking at actually. I'm not too comfortable with adding new stuff though as I'm not a lore buff and am not sure what to add. My biggest passion is for removing those god awful "conquer the world" conditions where you have to take landmarks, which is what Rakath has but apparently his is reasonable and I haven't played his campaign yet.

I was actually thinking of changing Khatep's Black Pyramid condition though, but I think it's cute that all TK's factions have to do that.

**Comment by BoyNamedHsu** [comment]
Oh yeah, another thing. In the readme for the framework you wrote
`Its name has to be the key of the faction, which can be checked in the table factions, available at Dave`
and I have no idea what Dave is. I had to use [this site](https://www.honga.net/totalwar/warhammer2/) to find the IDs

**Comment by msolefonte** [comment]
Not this looks like a job well done. Merged.

> Oh yeah, another thing. In the readme for the framework you wrote
> `Its name has to be the key of the faction, which can be checked in the table factions, available at Dave`
> and I have no idea what Dave is. I had to use [this site](https://www.honga.net/totalwar/warhammer2/) to find the IDs

DAVE, DAtabase Visual Editor, is the official tool released by CA to query and modify the databases. Whenever you see someone at Steam playing _Total War: WARHAMMER - Assembly Kit_, it is probably Dave.

**Comment by msolefonte** [comment]
Fuck, I completely forgot it. You did not update the changelog. Please create a `[2.4.1] Minor Update - TBA` entry in the `CHANGELOG.md` file with your changes.

**Comment by BoyNamedHsu** [comment]
> Not this looks like a job well done. Merged.
> 
> > Oh yeah, another thing. In the readme for the framework you wrote
> > `Its name has to be the key of the faction, which can be checked in the table factions, available at Dave`
> > and I have no idea what Dave is. I had to use [this site](https://www.honga.net/totalwar/warhammer2/) to find the IDs
> 
> DAVE, DAtabase Visual Editor, is the official tool released by CA to query and modify the databases. Whenever you see someone at Steam playing _Total War: WARHAMMER - Assembly Kit_, it is probably Dave.

Oh gotcha. I'm new to all this, so It sounded like you were referring to your friend or something haha. 

> Fuck, I completely forgot it. You did not update the changelog. Please create a `[2.4.1] Minor Update - TBA` entry in the `CHANGELOG.md` file with your changes.

No problem, I'll do it when I get the chance. It's 3;00 am where I live!

**Pull request merged**